### PR TITLE
Add subtract functionality and test

### PR DESCRIPTION
**Pull request by emalwani1**
Closes issue #2

### CODE REVIEW

File: pythoncalculator/subtract.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def subtract(x, y)
```
Comment (pablin992) [inline pythoncalculator/subtract.py:1]: Missing colon  and missing new line at end of file

File: tests/test_subtract.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from pythoncalculator import subtract
+
+
+def test_subtract():
+    assert subtract(1, 3) == -2
+
```
Comment (pablin992) [inline tests/test_subtract.py:6]: missing new line

File: pythoncalculator/subtract.py (hunk 1 above)
Comment (pablin992) [reply pythoncalculator/subtract.py:1]: Missing colon and missing new line

**Review by pablin992 — COMMENTED** [review]
please correct

**Review by pablin992 — CHANGES_REQUESTED** [review]
please correct --

File: tests/test_subtract.py (hunk 2 above)
Comment (emalwani1) [reply tests/test_subtract.py:6]: How to correct the mistake?

**Pull request merged**